tests/checker/test_noproxy.py

```diff
@@ -17,8 +17,11 @@
 """
 Test proxy handling.
 """
-import httpserver
-from test.test_support import EnvironmentVarGuard
+from .import httpserver
```

Comment:
I think PEP-8 wants this to be `from . import httpserver`.

Reply:
Done.